code/controllers/subsystem/atoms.dm

```diff
@@ -25,13 +30,16 @@ SUBSYSTEM_DEF(atoms)
 	initialized = INITIALIZATION_INNEW_REGULAR
 	return ..()
 
-/datum/controller/subsystem/atoms/proc/InitializeAtoms(list/atoms)
+/datum/controller/subsystem/atoms/proc/InitializeAtoms(list/atoms, list/atoms_to_return = null)
 	if(initialized == INITIALIZATION_INSSATOMS)
 		return
 
 	old_initialized = initialized
 	initialized = INITIALIZATION_INNEW_MAPLOAD
 
+	if (atoms_to_return)
+		created_atoms = list()
```

Comment:
```suggestion
		LAZYINITLIST(created_atoms)
```

Reply:
i dont understand what the difference is exactly but done